subprojects/gradle/room-config/src/main/kotlin/com/avito/android/plugin/incremental/SystemOutMessenger.kt

```diff
@@ -0,0 +1,31 @@
+package com.avito.android.plugin.incremental
+
+import javax.annotation.processing.Messager
+import javax.lang.model.element.AnnotationMirror
+import javax.lang.model.element.AnnotationValue
+import javax.lang.model.element.Element
+import javax.tools.Diagnostic
+
+internal class SystemOutMessenger : Messager {
+    override fun printMessage(kind: Diagnostic.Kind?, msg: CharSequence?) {
+        println("ROOM Messenger: $kind $msg")
```

Comment:
Do we really need this message in stdout?

Reply:
I have some doubts but decided that RoomProccessor may print some useful information

Reply:
Not actual. Move this check from this PR to another task.